pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -78,41 +93,63 @@ export default Vue.extend<Data, Methods, Computed, never>({
         this.$store.dispatch('snapshots/fetch');
       }, 1500);
     },
+    setCardsHeight() {
+      /**
+       * height = window height - page header - space reserved for snapshot event
+       */
+      this.cardsStyle = `height: ${ window?.innerHeight - 150 - (this.snapshotEvent ? 75 : 0) }px`;
+    },
+    toggleMainScrollbar(value: boolean) {
+      const main = document.getElementsByTagName('main')?.[0];
+
+      if (main) {
+        main.style.overflowY = value ? 'auto' : 'hidden';
+      }
+    },
   },
 });
 </script>
 
 <template>
   <div class="snapshots">
-    <Banner
+    <div
       v-if="snapshotEvent"
-      class="banner mb-20"
-      :color="snapshotEvent.result"
-      :closable="true"
-      @close="snapshotEvent=null"
+      class="event"
```

Comment:
We can eliminate a lot of the code in this PR if we rely on CSS to do something like this

```
  .event {
    position: sticky;
    top: 0;
  }
```

Reply:
You are right, `position: sticky;` + other minor css adjustments did the trick. Thanks for the hint.